### ISSUE TITLE
any experiences about the training loss ?

### ISSUE THREAD

**Issue by Papageno2** [Ma-Dan/keras-yolo4#17]
thanks for the nice repo.

 I am training yolov4 on coco dataset for only the first 7 classes with a split 70:30, 
any experiences or hints about the training loss when the model seems acceptable?

below is my traing log, seem a long time to train.
```shell
Train on 82801 samples, val on 35486 samples, with batch size 32.
Epoch 1/50

   1/2587 [..............................] - ETA: 28:56:58 - loss: 17721.8926
   2/2587 [..............................] - ETA: 16:33:45 - loss: 17243.4922
   3/2587 [..............................] - ETA: 12:32:28 - loss: 16760.1377
   4/2587 [..............................] - ETA: 10:10:04 - loss: 16299.9021
   5/2587 [..............................] - ETA: 9:24:35 - loss: 15857.3045 
   6/2587 [..............................] - ETA: 9:14:57 - loss: 15430.3268
   7/2587 [..............................] - ETA: 9:07:24 - loss: 15008.8902
   8/2587 [..............................] - ETA: 9:01:51 - loss: 14608.8591
   9/2587 [..............................] - ETA: 9:12:25 - loss: 14219.5904
  10/2587 [..............................] - ETA: 9:20:37 - loss: 13848.0055
  11/2587 [..............................] - ETA: 9:18:39 - loss: 13490.1777
  12/2587 [..............................] - ETA: 8:59:26 - loss: 13150.2271
  13/2587 [..............................] - ETA: 8:44:20 - loss: 12822.8066
  14/2587 [..............................] - ETA: 8:34:12 - loss: 12509.4747
  15/2587 [..............................] - ETA: 8:32:09 - loss: 12211.5387
  16/2587 [..............................] - ETA: 8:26:59 - loss: 11925.6479
  17/2587 [..............................] - ETA: 8:20:37 - loss: 11649.0334
  18/2587 [..............................] - ETA: 8:21:20 - loss: 11381.8880
  19/2587 [..............................] - ETA: 8:14:07 - loss: 11124.9040
  20/2587 [..............................] - ETA: 8:11:29 - loss: 10882.4443
  21/2587 [..............................] - ETA: 8:07:56 - loss: 10645.2164
  22/2587 [..............................] - ETA: 8:03:53 - loss: 10420.4502
  23/2587 [..............................] - ETA: 7:58:37 - loss: 10203.8660
  24/2587 [..............................] - ETA: 7:54:08 - loss: 9995.8066 
  25/2587 [..............................] - ETA: 7:50:39 - loss: 9794.8682
  26/2587 [..............................] - ETA: 7:48:39 - loss: 9599.6988
  27/2587 [..............................] - ETA: 7:45:55 - loss: 9411.8161
  28/2587 [..............................] - ETA: 7:42:36 - loss: 9234.4692
  29/2587 [..............................] - ETA: 7:39:51 - loss: 9060.8342
  30/2587 [..............................] - ETA: 7:35:15 - loss: 8894.1920
  31/2587 [..............................] - ETA: 7:33:45 - loss: 8733.4837
  32/2587 [..............................] - ETA: 7:31:51 - loss: 8577.9887
  33/2587 [..............................] - ETA: 7:29:49 - loss: 8425.8984
  34/2587 [..............................] - ETA: 7:27:43 - loss: 8280.7446
  35/2587 [..............................] - ETA: 7:26:30 - loss: 8139.6654
  36/2587 [..............................] - ETA: 7:26:01 - loss: 8004.0953
  37/2587 [..............................] - ETA: 7:26:13 - loss: 7872.3828
  38/2587 [..............................] - ETA: 7:24:52 - loss: 7746.5956
  39/2587 [..............................] - ETA: 7:23:28 - loss: 7624.6465
  40/2587 [..............................] - ETA: 7:29:19 - loss: 7506.7023
  41/2587 [..............................] - ETA: 7:35:16 - loss: 7391.7025
  42/2587 [..............................] - ETA: 7:34:34 - loss: 7280.5936
  43/2587 [..............................] - ETA: 7:31:58 - loss: 7172.5243
  44/2587 [..............................] - ETA: 7:29:56 - loss: 7067.7942
  45/2587 [..............................] - ETA: 7:30:26 - loss: 6965.9564
  46/2587 [..............................] - ETA: 7:31:08 - loss: 6867.1803
  47/2587 [..............................] - ETA: 7:34:20 - loss: 6771.7925
```

**Comment by github2016-yuan** [comment]
我在voc上训练的，loss一开始就是很大，1万多。后来我试着在yolov3上训练，也是这样的。但是在wins上，直接用darknet训练，就没有这种情况，loss一开始就几十。原因不明。

**Comment by Papageno2** [comment]
after 4 epoch, it reaches

```shell
1838/2587 [====================>.........] - ETA: 2:37:20 - loss: 65.4863
1839/2587 [====================>.........] - ETA: 2:37:08 - loss: 65.4886
1840/2587 [====================>.........] - ETA: 2:36:56 - loss: 65.4875
1841/2587 [====================>.........] - ETA: 2:36:43 - loss: 65.4870
1842/2587 [====================>.........] - ETA: 2:36:30 - loss: 65.4876
```

**Comment by github2016-yuan** [comment]
差不多,我一个epoch后loss能达到160多,但是这是有问题.loss正常从几十开始.可能标注数据的问题,但是我用了下面这2种 ,一样的loss巨大.
第一种:
/VOCdevkit/VOC2007/JPEGImages/000012.jpg 156,97,351,270,6

第二种:
/VOCdevkit/VOC2007/JPEGImages/000016.jpg  0.4638835 0.5425 0.927767 0.9137500000000001 6
所以很奇怪,我看别人都没有遇到这种情况.

**Comment by Papageno2** [comment]
> 差不多,我一个epoch后loss能达到160多,但是这是有问题.loss正常从几十开始.可能标注数据的问题,但是我用了下面这2种 ,一样的loss巨大.
> 第一种:
> /VOCdevkit/VOC2007/JPEGImages/000012.jpg 156,97,351,270,6
> 
> 第二种:
> /VOCdevkit/VOC2007/JPEGImages/000016.jpg 0.4638835 0.5425 0.927767 0.9137500000000001 6
> 所以很奇怪,我看别人都没有遇到这种情况.

I think yolo format bbox should be absolute pixel rather than relative scale ?

**Comment by github2016-yuan** [comment]
是的,绝对像素值.你试过keras-yolov3吗?
